src/main/java/com/treasure/hunt/strategy/hider/impl/AngleHintStat.java

```diff
@@ -0,0 +1,49 @@
+package com.treasure.hunt.strategy.hider.impl;
+
+import com.treasure.hunt.strategy.hint.Hint;
+import com.treasure.hunt.strategy.hint.impl.AngleHint;
+import com.treasure.hunt.utils.JTSUtils;
+import javafx.util.Pair;
+import lombok.Getter;
+import lombok.Setter;
+import org.locationtech.jts.algorithm.Angle;
+import org.locationtech.jts.geom.Coordinate;
+
+public class AngleHintStat {
```

Comment:
So I have no idea what this is maybe because u are using abbreviatiosn for the class name what does this do

Reply:
Thought this was self explanatory, Its a class which saves Stats about an AngleHint